channels_yroom/worker.py

```diff
@@ -96,3 +131,4 @@ async def shutdown_worker(self, signal, loop):
     async def receive_from_worker(self, message):
         assert message["type"] == "shutdown.complete"
         self.shutdown_event.set()
+        self.shutdown_event.set()
```

Comment:
Need to call here twice?

Reply:
Nope, this is https://github.com/charliermarsh/ruff-vscode/issues/158 :(